src/AdminSettings.vue

```diff
@@ -20,28 +20,41 @@
 -->
 
 <template>
-	<NcSettingsSection :title="t('announcementcenter', 'Announcements')">
-		<NcSettingsSelectGroup id="announcementcenter_admin_group"
-			v-model="adminGroups"
-			:label="t('announcementcenter', 'These groups will be able to post announcements.')"
-			@input="updateGroups" />
+	<NcSettingsSection :name="t('announcementcenter', 'Announcements')">
+		<div>
```

Comment:
Anything but this div is changed whitespaces? Could it be avoided to keep the diff small and reduce conflict with other work?

Reply:
Indentation, attribute `:title` becomes  `:name` for `NcSettingsSection`, and the label for `NcSettingsSelectGroup` is extracted.